src/components/DataMessenger/DataMessenger.js

```diff
@@ -241,7 +258,12 @@ export default class DataMessenger extends React.Component {
         this.clearMessages()
       }
       if (this.state.messages.length === 0) {
-        this.setintroMessages()
+        const { isLoadedTopics } = this.state
+        if (isLoadedTopics) {
+          this.setintroMessages()
```

Comment:
Both of these blocks have the same this.setintroMessages(), why have an if statement at all?

Also can we change this to setIntroMessages with the capital "i"   must have been a typo the first time